en_us.lang

```diff
@@ -157,200 +157,200 @@ gui.xaero_transfer_all=Transfer All Waypoints
 gui.xaero_box_entity_amount=The limit per each entity category.
-gui.xaero_box_distance=Which ingame waypoints to show the distance to for.
-gui.xaero_box_always_distance=Display distance even when the waypoint is very close. This does not override the "Distance To WP" option.
-gui.xaero_box_arrow_scale=Scale for the arrow used in the non-rotating variant of the minimap and some other cases.
-gui.xaero_box_arrow_color=Color of the arrow used in the non-rotating variant of the minimap and some other cases.
-gui.xaero_box_lock_player_heads=Display player heads instead of coloured dots even if the TAB key is not pressed.
-gui.xaero_box_cave_zoom=How much to zoom further in when the cave maps mode is active.
-gui.xaero_box_entity_depth=Display entities darker depending on their Y level relative to you.
-gui.xaero_box_height_limit=Maximum Y level distance for entities until they are no longer displayed.
-gui.xaero_box_compass_over_wp=Render compass letters (N, E, S, W) over the on-map waypoints.,
-gui.xaero_toggle_map_waypoints=Toggle On-map Waypoints
-gui.xaero_entity_nametags=Entity Name Tags
+gui.xaero_box_distance=Specifies, for which in-game waypoints to show the distance to.
+gui.xaero_box_always_distance=Display distance even when the waypoint is very close. This setting does not override the "Distance to WP" option.
+gui.xaero_box_arrow_scale=Scale of the arrow used in the non-rotating variant of the minimap and several other cases.
+gui.xaero_box_arrow_color=Color of the arrow used in the non-rotating variant of the minimap and several other cases.
+gui.xaero_box_lock_player_heads=Display player heads instead of colored dots even if the Tab key is not pressed.
+gui.xaero_box_cave_zoom=How much to zoom further in the minimap when the cave mode is active.
+gui.xaero_box_entity_depth=Display entities darker on the minimap depending on their Y level relative to you.
+gui.xaero_box_height_limit=Maximum Y-level distance for entities until they are no longer displayed on the minimap.
+gui.xaero_box_compass_over_wp=Render compass letters (N, E, S, W) over the on-map waypoints.
+gui.xaero_toggle_map_waypoints=Toggle On-Map Waypoints
+gui.xaero_entity_nametags=Entities With Name Tags
 gui.xaero_minimap_text_align=Info Text Alignment
 gui.xaero_center=Center
 gui.xaero_left=Left
 gui.xaero_right=Right
-gui.xaero_show_angles=Show View Angles
-gui.xaero_box_entity_nametags=Display entity name under the dot for entities that have a name tag.
-gui.xaero_box_teleport_default_command=The command used for waypoints teleportation if a server-specific command is not set in the waypoints menu Options screen.
-gui.xaero_waypoints_edit_mode_individually=One-by-one
-gui.xaero_waypoints_edit_mode_all=All
+gui.xaero_show_angles=Display Facing Angles
```

Comment:
I think the original is better but maybe you can explain why it's not ¯\\\_(ツ)\_/¯

Reply:
Because most likely view angle or angle of view are not what you meant: https://en.wikipedia.org/wiki/Angle_of_view. Vanilla Minecraft uses word "facing" as well. You can look e.g. at the F3 screen or at the /tp command syntax to see it.

Reply:
Oh, ok. I wasn't really trying to use a term there. There's a view and it's angled lol. I guess your variant is better then.